sql/core/src/test/resources/sql-tests/inputs/mode.sql

```diff
@@ -0,0 +1,112 @@
+-- Test data.
+CREATE OR REPLACE TEMPORARY VIEW basic_pays AS SELECT * FROM VALUES
+('Diane Murphy','Accounting',8435),
+('Mary Patterson','Accounting',9998),
+('Jeff Firrelli','Accounting',8992),
+('William Patterson','Accounting',8870),
+('Gerard Bondur','Accounting',11472),
+('Anthony Bow','Accounting',6627),
+('Leslie Jennings','IT',8113),
+('Leslie Thompson','IT',5186),
+('Julie Firrelli','Sales',9181),
+('Steve Patterson','Sales',9441),
+('Foon Yue Tseng','Sales',6660),
+('George Vanauf','Sales',10563),
+('Loui Bondur','SCM',10449),
+('Gerard Hernandez','SCM',6949),
+('Pamela Castillo','SCM',11303),
+('Larry Bott','SCM',11798),
+('Barry Jones','SCM',10586)
+AS basic_pays(employee_name, department, salary);
+
+SELECT
+  mode() WITHIN GROUP (ORDER BY col),
+  mode() WITHIN GROUP (ORDER BY col DESC)
```

Comment:
hmm, do we need a new golden file? I think `group-by.sql` tests the functionality of `mode` well, we just need to add a few more cases for `WITHIN GROUP`, as well as some negative cases. Or we should move all `mode` tests to this new file.

Reply:
I tend to concentrate on the same file.